integrationTests/i3CRVlvUSDCurvePool.ts

```diff
@@ -0,0 +1,82 @@
+import { ethers } from "hardhat";
+import { abilvUSD, abilvUSD3CRVPool, abiUSDC, abiZap } from "../test/ABIs";
+import { addressZap, address3CRV, addressUSDC, defaultBlockNumber, helperResetNetwork, address3CRVlvUSDPool } from "../test/MainnetHelper";
+import { impersonateAccount, fundAccount, stopImpersonate, addresslvUSDToken } from "./IntegrationTestContext";
+
+const About4700 = "0x1000000000000000000";
+
+const addr1 = "0x55fe002aeff02f77364de339a1292923a15844b8"; // Circles address
+const addresslvUSDMinter = "0x42208d094776c533ee96a4a57d50a6ac04af4aa2";
+const addresslvUSDAdmin = "0x7246dd11320eee513cefe5f50e8be2d28fb06426";
+
+let contractlvUSDToken: Contract;
+let contractUSDC: Contract;
+let contractlvUSD3CRVPool: Contract;
+let signerlvUSDAdmin: Signer;
+let signerlvUSDMinter: Signer;
+let signerAddr1: Signer;
+
+describe("3CRV/lvUSD curve pool test suit", function () {
+    before(async function () {
+        await helperResetNetwork(defaultBlockNumber);
+
+        // grab lvUSD token contract
+        contractlvUSDToken = await ethers.getContractAt(abilvUSD, addresslvUSDToken);
+
+        // grab lvUSD admin address
+        signerlvUSDAdmin = await impersonateAccount(addresslvUSDAdmin);
+        fundAccount(addresslvUSDAdmin, About4700);
+
+        // grab lvUSD minter address
+        signerlvUSDMinter = await impersonateAccount(addresslvUSDMinter);
+        fundAccount(addresslvUSDMinter, About4700);
+
+        // grab random test address
+        signerAddr1 = await impersonateAccount(addr1);
+        fundAccount(addr1, About4700);
+    });
+
+    after(async function () {
+        await stopImpersonate(addresslvUSDMinter);
+        await stopImpersonate(addresslvUSDAdmin);
+        await stopImpersonate(addr1);
+    });
+
+    it("Add liquidity to 3CRV/lvUSD pool", async function () {
+        const tenK18Decimal = ethers.utils.parseUnits("10000", 18);
+        const tenK6Decimal = ethers.utils.parseUnits("10000", 6);
+
+        // set lvUSD mint address
+        await contractlvUSDToken.connect(signerlvUSDAdmin).setMintDestination(addr1);
+        // mint a bit lvUSD
+        await contractlvUSDToken.connect(signerlvUSDMinter).mint(tenK18Decimal);
+
+        // grab some USDC - addr1 is Circle's so it has a lot of USDC need to verify though
+        contractUSDC = await ethers.getContractAt(abiUSDC, addressUSDC);
+        expect(await contractUSDC.balanceOf(addr1)).to.be.gte(tenK6Decimal);
+
+        // approve Zap contract to grab lvUSD and USDC from addr1
+        await contractlvUSDToken.connect(signerAddr1).approve(addressZap, tenK18Decimal);
+        await contractUSDC.connect(signerAddr1).approve(addressZap, tenK6Decimal);
+
+        // get 3CRV/lvUSD contract
+        contractlvUSD3CRVPool = await ethers.getContractAt(abilvUSD3CRVPool, address3CRVlvUSDPool);
+
+        // grab the "before" balances so we can check they increase after adding liquidity
+        const balancelvUSD = await contractlvUSD3CRVPool.balances(0);
```

Comment:
Change name to balancePoolLvUSD and same for balanceUSDC?

Reply:
ack